RELEASING.md

```diff
@@ -16,16 +16,32 @@ Things to do before issuing a new release:
 
 * Test the standalone release (this should be done on windows as well as linux):
 1. Unpack the distribution zip or tar.gz
-2. Start and stop the server (run.sh and stop.sh) for each configuration
+2. Start and stop the server
 3. Run the examples (follow the instructions under examples/index.html)
 5. Check the manuals have been created properly
 6. Check the javadocs are created correctly (including the diagrams)
 
-* If every thing is successful.  Follow the Apache guide (http://www.apache.org/dev/publishing-maven-artifacts.html) to build and publish artifacts to Nexus and send out a release vote.
```

Comment:
I think we should keep that reference...

At least keep this as a note: this is based on http://www.apache.org/dev/publishing-maven-artifacts.html